packages/client/src/components/Categories/CategoriesAndSearchBarAndSortBy.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { CategoryButton } from './CategoryButton';
+import { Box } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useMoviesAndCategoriesContext } from '../../context/FetchCategoriesAndMovies';
+import { SearchBar } from '../SearchBar/SearchBar';
+import { CustomSelect } from '../Select/CustomSelect';
+
+export const CategoriesAndSearchBarAndSortBy = () => {
+  const { state, dispatch } = useMoviesAndCategoriesContext();
+  const theme = useTheme();
+  const handleOnClick = (catId) => {
```

Comment:
just try to name it better, and include the category in the function name, so that it gives more clarity